### ISSUE TITLE
Flow Chart Accessibility

### ISSUE THREAD

**Issue by sergiobaptista** [TBD54566975/tbdex-protocol#65]
The flow charts on the following pages have been identified as needing accessibility labels applied: (though all flow charts should be made accessible)
* https://github.com/TBD54566975/tbdex-protocol/tree/main/pfi-mock-impl#readme
* https://github.com/TBD54566975/tbdex-protocol/tree/main/lib#readme

See documentation re: Mermaid's accessibility options: https://mermaid-js.github.io/mermaid/#/accessibility?id=accessibility
